capture_service/device_mgr.h

```diff
@@ -70,12 +70,14 @@ class AndroidDevice
 
     struct PackageListOptions
```

Comment:
would it be better to use enum here? all 3 options seems to be mutually exclusive?